src/client/blocks.js

```diff
@@ -1131,10 +1131,10 @@ InputSlotMorph.prototype.rpcNames = function () {
     var rpcs = JSON.parse(this.getURL('/rpc')),
         dict = {},
         name;
-
-    for (var i = rpcs.length; i--;) {
+    for (var i = 0; i < rpcs.length; i++) {
         name = rpcs[i].replace('/', '');
-        dict[name] = name;
+        let label = name.charAt(0).toUpperCase() + name.slice(1);
```

Comment:
Can't use ES2015 on the client code atm.

Also, I am not sure if this is the best way to address the naming convention. It won't really convert it from hyphenated to camelcase, etc. I think it would be best to enforce this in the tests probably and just manually rename them.